packages/fluentui/react-builder/src/components/DropSelector.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { EventListener } from '@fluentui/react-component-event-listener';
+import { isBrowser } from '@fluentui/react-northstar';
+import FiberNavigator from '../../../react-northstar/src/components/Debug/FiberNavigator';
+import { fiberNavFindJSONTreeElement, jsonTreeFindParent } from '../config';
+import { JSONTreeElement } from './types';
+
+export type DropSelectorProps = {
+  onDropPositionChange: (dropParent: JSONTreeElement, dropIndex: number) => void;
+  filter?;
+  jsonTree: JSONTreeElement;
+  mountDocument?: Document;
+};
+
+type InElementPosition = 'top' | 'right' | 'bottom' | 'left' | 'center';
+
+// left/right wins over top/bottom
+const getInElementPosition: (
+  elementRect: { top: number; left: number; width: number; height: number },
```

Comment:
```suggestion
  elementRect: ClientRect,
```
??

Reply:
`ClientRect` does not have editable `width` and `height` :-(